### ISSUE TITLE
If i set auto-start = false, How do I manually consume information

### ISSUE THREAD

**Issue by riritack** [majusko/pulsar-java-spring-boot-starter#248]
If i set auto-start = false, How do I manually consume information？

**Comment by majusko** [comment]
I don't think that was the initial idea. This feature was proposed to switch consumption off on some environments as "dev" and it wasn't meant to start these consumers manually. However, it would be possible and I think quite simple to do so we can implement a manual consumer start if you are interested.
